maxim/logger/openai/utils.py

```
@@ -122,34 +122,77 @@ def parse_completion(
+
+        # Parse each chunk
+        parsed_chunks = [OpenAIUtils.parse_stream_response(chunk) for chunk in chunks]
+
+        if parsed_chunks:
+            last_chunk = parsed_chunks[-1]
+            combined_content = "".join(
+                [
+                    choice.get("delta", {}).get("content", "")
+                    for chunk in parsed_chunks
+                    for choice in chunk.get("choices", [])
+                ]
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)